buildpacks/dotnet/src/launch_process.rs

```diff
@@ -23,23 +23,36 @@ fn project_launch_process(solution: &Solution, project: &Project) -> Option<Proc
     }
     let relative_executable_path = relative_executable_path(solution, project);
 
+    let command = build_command(&relative_executable_path, project.project_type);
+
+    Some(ProcessBuilder::new(project_process_type(project), ["bash", "-c", &command]).build())
+}
+
+/// Constructs the shell command for launching the process
+fn build_command(relative_executable_path: &Path, project_type: ProjectType) -> String {
+    let parent_dir = relative_executable_path
+        .parent()
+        .expect("Executable path should always have a parent directory")
+        .to_str()
+        .expect("Path should be valid UTF-8");
+
+    let file_name = relative_executable_path
+        .file_name()
+        .expect("Executable path should always have a file name")
+        .to_str()
+        .expect("Path should be valid UTF-8");
+
     let mut command = format!(
         "cd {}; ./{}",
```

Comment:
Now that the CNB spec supports the `working-dir` feature, could this `cd` be replaced with that?
https://github.com/buildpacks/spec/blob/main/buildpack.md#launchtoml-toml